### ISSUE TITLE
sbccpatterns.com

### ISSUE THREAD

**Issue by Alex-302** [realodix/AdBlockID#1481]
### Issue URL (Incorrect Blocking)
 [https://www.sbccpatterns.com/collections/all](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.sbccpatterns.com%2Fcollections%2Fall)
### Comment
> about half the product images are missing with no alt text or icon or error image.  Clicking through to a detail page doesn't get the affected images to load.

Caused by `/sbcc-$image` from `AdBlockID`.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  ![Screenshot 1](https://cdn.adguard.info/sitereports/7wk46ecdglgkc8cwck80sgw480ksk45o1ykuag09s0.jpg?nc=1)
</details>
<details>
  <summary>Screenshot 2</summary>

  ![Screenshot 2](https://cdn.adguard.info/sitereports/4q31fm8pdhk4go80wgc004cwkc4kokkmmfy9ggf74s.jpg?nc=1)
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.1.57
Browser: | Firefox
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Other:</b><br/>Filter unblocking search ads and self-promotion<br/><br/><details><summary><b>Language-specific:</b></summary>AdGuard Turkish, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard Ukrainian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese, <br/>Bulgarian list, <br/>EasyList Czech and Slovak, <br/>EasyList Hebrew, <br/>EasyList Italy, <br/>AdBlockID, <br/>EasyList Thailand, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>List-KR, <br/>Adblock List for Finland, <br/>Polish Anti Adblock Filters, <br/>Frellwit's Swedish Filter, <br/>Dandelion Sprout's Nordic Filters</details>
Other extensions used: | none
